pkg/deploy/config_test.go

```diff
@@ -51,6 +51,11 @@ func TestConfigurationFieldParity(t *testing.T) {
 }
 
 func TestMergeConfig(t *testing.T) {
+	databaseAccountName := "primary accountname"
```

Comment:
for tests always use the field name as variable names: `databaseAccountName := "databaseAccountName"`. Makes it easier to read when they fail.

Reply:
(where possible, some fields might not be possible)